Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM debian:buster-slim
+
+# Common tools
+RUN apt update && apt install -y ca-certificates curl gnupg make zip unzip wget apt-utils  \
```

Comment:
if i understand correctly, we are building this image every time we run the pipeline. If that's the case we are also updating the image every time and changing the build environment. I would suggest creating a base image of our own and use that to keep the environment consistent.

Reply:
This will be used only for development environment. It is still not decided if we use it for CI/CD (but can be used)

What's the benefit of using our base image?

Reply:
If it's just for dev env, this is okay. My concern is if it was used for pipeline, then we are changing the container every time we do update..